taskcluster/linux-opt-base.tyml

```diff
@@ -39,11 +39,12 @@ payload:
     - $let:
         extraSystemSetup: { $eval: strip(str(build.system_setup)) }
         extraSystemConfig: { $eval: strip(str(build.system_config)) }
+        installGitlfs: { $eval: strip(str(build.git_lfs)) }
       in: >
         apt-get -qq update && apt-get -qq -y install git pixz wget pkg-config libsox-dev && ${extraSystemSetup} &&
         adduser --system --home /home/build-user build-user &&
         cd /home/build-user/ &&
-        echo -e "#!/bin/bash\nset -xe\nenv && id && (wget -O - $TENSORFLOW_BUILD_ARTIFACT | pixz -d | tar -C /home/build-user/ -xf - ) && git clone --quiet ${event.head.repo.url} ~/DeepSpeech/ds/ && cd ~/DeepSpeech/ds && git checkout --quiet ${event.head.sha} && ln -s ~/DeepSpeech/ds/native_client/ ~/DeepSpeech/tf/native_client" > /tmp/clone.sh && chmod +x /tmp/clone.sh &&
+        echo -e "#!/bin/bash\nset -xe\n ${installGitlfs} && env && id && (wget -O - $TENSORFLOW_BUILD_ARTIFACT | pixz -d | tar -C /home/build-user/ -xf - ) && git clone --quiet ${event.head.repo.url} ~/DeepSpeech/ds/ && cd ~/DeepSpeech/ds && git checkout --quiet ${event.head.sha} && ln -s ~/DeepSpeech/ds/native_client/ ~/DeepSpeech/tf/native_client" > /tmp/clone.sh && chmod +x /tmp/clone.sh &&
```

Comment:
I'm just wondering, since `OSX` builds does not have that and they do go green, maybe that should just be moved to `test-linux-opt-base` instead ?